### ISSUE TITLE
Proposal to transfer UlisesGascon/nodejs-distribution-system-monitoring repo into the Node.js organization

### ISSUE THREAD

**Issue by UlisesGascon** [nodejs/admin#821]
### TL;DR:

I want to migrate my repository [UlisesGascon/nodejs-distribution-system-monitoring](https://github.com/UlisesGascon/nodejs-distribution-system-monitoring) to the Node.js org (as `nodejs/binaries-integrity-checker`), in order to help the website and build teams to monitor the Node.js Binaries during the R2 Migration.

cc: @nodejs/website @nodejs/build

### Notes

- I will need to be admin in this repo (if possible) as I need to change the repo settings (Tokens, branch protection rules, etc...). 
- I will prepare the repo for the migration soon (license CoC, etc.. ), it will be ready before the migration is executed.

### Context

More details, motivation, context and discussions can be found here: https://github.com/nodejs/build/issues/3469.

**Comment by mhdawson** [comment]
+1 from me

**Comment by trivikr** [comment]
What's blocking Node.js Binaries Integrity Checker migration to Node.js org?

**Comment by trivikr** [comment]
Steps to follow before migration: https://github.com/UlisesGascon/nodejs-distribution-system-monitoring/issues/5

**Comment by mapangasar** [comment]
.github/workflows
